modules/nation_notes/nn_library_options.inc

```diff
@@ -3004,13 +3001,13 @@ function expand_collapse_widget($title, $label, $buttonLabel, $buttonLink, $butt
 
 //billing_facility fuction will give the dropdown list which contain billing faciliies.
 function billing_facility($name,$select){
-	$qsql = sqlStatement("SELECT id, name FROM facility WHERE billing_location = 1");
-		echo "   <select id='".htmlspecialchars($name, ENT_QUOTES)."' name='".htmlspecialchars($name, ENT_QUOTES)."'>";
-			while ($facrow = sqlFetchArray($qsql)) {
-				$selected = ( $facrow['id'] == $select ) ? 'selected="selected"' : '' ;
-				 echo "<option value=".htmlspecialchars($facrow['id'],ENT_QUOTES)." $selected>".htmlspecialchars($facrow['name'], ENT_QUOTES)."</option>";
-				}
-			  echo "</select>";
+    $qsql = sqlStatement("SELECT id, name FROM facility WHERE billing_location = 1");
+        echo "   <select id='".htmlspecialchars($name, ENT_QUOTES)."' name='".htmlspecialchars($name, ENT_QUOTES)."'>";
+            while ($facrow = sqlFetchArray($qsql)) {
+                $selected = ( $facrow['id'] == $select ) ? 'selected="selected"' : '' ;
+                 echo "<option value=".htmlspecialchars($facrow['id'],ENT_QUOTES)." $selected>".htmlspecialchars($facrow['name'], ENT_QUOTES)."</option>";
+                }
+              echo "</select>";
 }
 
```

Comment:
No code changes found in this file.  Thousands of lines changed...with no change.